diff --git a/src/libnfc-utils/src/ConfigPathProvider.cc b/src/libnfc-utils/src/ConfigPathProvider.cc
index 7927880..c7c0316 100644
--- a/src/libnfc-utils/src/ConfigPathProvider.cc
+++ b/src/libnfc-utils/src/ConfigPathProvider.cc
@@ -83,17 +83,17 @@ string ConfigPathProvider::getFilePath(FileType type) {
         << StringPrintf("%s: enter FileType:0x%02x", __func__, type);
     switch (type) {
     case VENDOR_NFC_CONFIG: {
-        string path = "//usr//local//etc//libnfc-nxp.conf";
+        string path = "//etc//libnfc-nxp.conf";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case VENDOR_ESE_CONFIG: {
-        string path = "//usr//local//etc//libese-nxp.conf";
+        string path = "//etc//libese-nxp.conf";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case SYSTEM_CONFIG: {
-        const vector<string> searchPath = { "//usr/local//etc//" };
+        const vector<string> searchPath = { "//etc//" };
         for (string path : searchPath) {
             addEnvPathIfAvailable(path);
             path.append("libnfc-nci.conf");
@@ -105,12 +105,12 @@ string ConfigPathProvider::getFilePath(FileType type) {
         return "";
     } break;
     case RF_CONFIG: {
-        string path = "//usr//local//etc//libnfc-nxp_RF.conf";
+        string path = "//etc//libnfc-nxp_RF.conf";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case TRANSIT_CONFIG: {
-        string path = "//usr/local//etc//libnfc-nxpTransit.conf";
+        string path = "//etc//libnfc-nxpTransit.conf";
         addEnvPathIfAvailable(path);
         return path;
     } break;
@@ -121,22 +121,22 @@ string ConfigPathProvider::getFilePath(FileType type) {
         return nfc_storage_path;
     } break;
     case FIRMWARE_LIB: {
-        string path = "//usr/local//etc//libsn100u_fw.dll";
+        string path = "//etc//libsn100u_fw.dll";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case CONFIG_TIMESTAMP: {
-        string path = "//usr//local//etc//libnfc-nxpConfigState.bin";
+        string path = "//etc//libnfc-nxpConfigState.bin";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case RF_CONFIG_TIMESTAMP: {
-        string path = "//usr//local//etc//libnfc-nxpRFConfigState.bin";
+        string path = "//etc//libnfc-nxpRFConfigState.bin";
         addEnvPathIfAvailable(path);
         return path;
     } break;
     case TRANSIT_CONFIG_TIMESTAMP: {
-        string path = "//usr//local//etc//libnfc-nxpTransitConfigState.bin";
+        string path = "//etc//libnfc-nxpTransitConfigState.bin";
         addEnvPathIfAvailable(path);
         return path;
     } break;
