From 99f4d668cef4e7af54fef4ef49631385dacbf59d Mon Sep 17 00:00:00 2001
From: Lukas Janik <janik@acrios.com>
Date: Thu, 6 Feb 2025 11:56:22 +0100
Subject: [PATCH 3/3] pn5xx_i2c.c: Fix types of probe() and remove()

---
 pn5xx_i2c.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/pn5xx_i2c.c b/pn5xx_i2c.c
index 16471fa..388c3f5 100644
--- a/pn5xx_i2c.c
+++ b/pn5xx_i2c.c
@@ -514,8 +514,7 @@ static int pn54x_get_pdata(struct device *dev,
  static int __devinit pn54x_probe(struct i2c_client *client,
 		const struct i2c_device_id *id)
 #else
-static int pn54x_probe(struct i2c_client *client,
-		const struct i2c_device_id *id)
+static int pn54x_probe(struct i2c_client *client)
 #endif
 {
 	int ret;
@@ -696,7 +695,7 @@ err_ven:
 #ifdef KERNEL_3_4_AND_OLDER
 static int __devexit pn54x_remove(struct i2c_client *client)
 #else
-static int pn54x_remove(struct i2c_client *client)
+static void pn54x_remove(struct i2c_client *client)
 #endif
 {
 	struct pn54x_dev *pn54x_dev;
@@ -719,8 +718,9 @@ static int pn54x_remove(struct i2c_client *client)
 	regulator_put(pn54x_dev->sevdd_reg);
 
 	kfree(pn54x_dev);
-
-	return 0;
+#ifdef KERNEL_3_4_AND_OLDER
+	return 0
+#endif
 }
 
 /*
-- 
2.48.1

